packages/core/src/Engine.ts

```diff
@@ -58,6 +59,10 @@ export class Engine extends EventDispatcher {
   protected _canvas: Canvas;
   private _resourceManager: ResourceManager = new ResourceManager(this);
   private _sceneManager: SceneManager = new SceneManager(this);
+  /**
+   * Physics manager of Engine.
+   */
+  readonly physicsManager: PhysicsManager = new PhysicsManager(this);
```

Comment:
The public field is at the top of all instance fields.
refer to https://github.com/oasis-engine/engine/issues/352